### PR TITLE
In Sub _merge_msg Content-Disposition: attachement in multipart-messa…

### PR DESCRIPTION
**Pull request by pveith**
This should resolve Issue 409 as far as i tested it (Invitations and attachment files with special characters). Further Information see there.

### CODE REVIEW

File: src/lib/Sympa/Message.pm [hunk 1]
```diff
@@ -1262,6 +1262,13 @@ sub _merge_msg {
         return $entity;
     }
 
+    # Check for attchment-part, which should not be changed
+    my $cdisposition = $entity->head->mime_attr('Content-Disposition');
+    if ($cdisposition and $cdisposition == 'attachment') {
```
Comment (ikedas) [inline src/lib/Sympa/Message.pm]: eq should be used.

Comment (ikedas) [reply src/lib/Sympa/Message.pm]: and both upper/lowercase values should be considered.

**Comment by pveith** [comment]
I installed the PR on our production sympa server, which now emits quite a few notices in the log. Everything works as expected though, as far as i can see; i would be inclined to drop the log notice.

**Comment by ikedas** [comment]
@pveith, if you haven't experienced problem, I'll merge this pr.

**Comment by pveith** [comment]
No Problems detected this week. After using it in our production server for a week, i think the PR works just as expected.

**Comment by ikedas** [comment]
Merged. Thanks!

**Pull request merged**